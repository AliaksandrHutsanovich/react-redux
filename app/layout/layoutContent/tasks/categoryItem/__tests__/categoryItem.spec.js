import React from 'react';
import { shallow } from 'enzyme';
import CategoryItem from '../categoryItem';

describe('Unit test of category item component', () => {
  const Component = shallow(<CategoryItem title="A title of category" />);
  it('full render test', () => {
    expect(Component).toMatchSnapshot();
  });
});
