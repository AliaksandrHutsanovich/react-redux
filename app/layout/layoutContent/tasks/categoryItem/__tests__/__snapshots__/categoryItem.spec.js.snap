// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test of category item component full render test 1`] = `
<Memo(CategoryItem)
  title="A title of category"
>
  <p
    className="itemText"
  >
    A title of category
  </p>
</Memo(CategoryItem)>
`;
